modules/ROOT/pages/api-instance-landing-page.adoc

```diff
@@ -28,4 +28,6 @@ To manage API instances in API Manager:
 ** xref:manage-versions-instances-concept.adoc[Manage API Versions] - After you create an API in Anypoint Platform, you configure the version details for the API and then publish it to Anypoint Exchange (Exchange). The API version resides in Exchange, and not in API Manager. However, you then manage this API version from API Manager.
 . Explore advanced use cases for API instances:
 +
-xref:configure-multiple-credential-providers.adoc[Configure Multiple Client Providers] - You can use multiple client providers, such as OpenAM and PingFederate, to help you enforce security and regulations in your business organization. These client providers enable you to secure your operational data, including client credentials and access tokens.
+* xref:configure-multiple-credential-providers.adoc[Configure Multiple Client Providers] - You can use multiple client providers, such as OpenAM and PingFederate, to help you enforce security and regulations in your business organization. These client providers enable you to secure your operational data, including client credentials and access tokens.
++
+* xref:govern-api-instances.adoc[Govern API Instances] - You can govern API instances by using the API Governance integration with API manager. API instance governance validation report details enable you to ensure that all your API instances follow the guidelines set out by governance rulesets. 
```

Comment:
They don't choose to use the APIG/APIM integration, so I suggest rephrasing this something like the following. I'd keep it simple here and rely on the link to the Govern API Instances page for more info.

You can govern API instances using Anypoint API Governance. View the governance validation report for governed instances to identify conformance issues and take action to fix them.